src/main.js

```diff
@@ -64,9 +64,11 @@ define(['storage','websockets'], function(storage, ws) {
     var container = document.getElementById(gmi.gameContainerId);
     var wrapper = document.createElement("div");
     var inner = document.createElement("div");
+    var inner2 = document.createElement("div");
     wrapper.className = "wrapper";
     wrapper.tabIndex = -1;
     inner.className = "inner";
+    inner2.className = "inner2";
```

Comment:
why do you need an inner2?

Reply:
oops that inner2 stuff was just testing, removed it locally but guess it didn't get committed, good spot.